<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SJN3UA7" failures="1" tests="4" name="Test_01" time="169.249" errors="0" timestamp="2021-09-19T19:42:15 IST">
  <testcase classname="com.qa.testScript.TC_OrangeHrm_001" name="Recruitment" time="19.969"/>
  <testcase classname="com.qa.testScript.TC_OrangeHrm_002" name="addCustomer" time="8.589"/>
  <testcase classname="com.qa.testScript.TC_OrangeHrm_011" name="addEmployee" time="25.471">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.qa.testScript.TC_OrangeHrm_011.addEmployee(TC_OrangeHrm_011.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- addEmployee -->
  <testcase classname="com.qa.testScript.TC_OrangeHrm_011" name="getEmployeeRecord" time="22.32"/>
</testsuite> <!-- Test_01 -->
